django.db.models.base.model_unpickle missing

apologies for cutting out the template -- most of the fields weren't relevant to my issue

I'm not sure what the correct typing of this is (I'm not all that familiar with django) https://github.com/django/django/blob/de2c2127b66e77a034c01c81753c5c08e651a5b4/django/db/models/base.py#L2464-L2471

but any access of this is giving an `[attr-defined]` error in mypy:

```python
django.db.models.base.model_unpickle
```

```
error: Module has no attribute "model_unpickle"  [attr-defined]
```

--- a/django-stubs/db/models/base.pyi
+++ b/django-stubs/db/models/base.pyi
@@ -1,4 +1,4 @@
-from typing import Any, Collection, Dict, Iterable, List, Optional, Sequence, Set, Tuple, Type, TypeVar
+from typing import Any, Collection, Dict, Iterable, List, Optional, Sequence, Set, Tuple, Type, TypeVar, Union
 
 from django.core.checks.messages import CheckMessage
 from django.core.exceptions import MultipleObjectsReturned as BaseMultipleObjectsReturned
@@ -65,3 +65,5 @@ class Model(metaclass=ModelBase):
     @classmethod
     def check(cls, **kwargs: Any) -> List[CheckMessage]: ...
     def __getstate__(self) -> dict: ...
+
+def model_unpickle(model_id: Union[Tuple[str, str], type[Model]]) -> Model: ...

